metals/src/main/scala/scala/meta/internal/metals/debug/DebugProtocol.scala

```diff
@@ -221,6 +222,7 @@ object DebugProtocol {
 
   def parse[A: ClassTag](params: Any): Try[A] = {
     params match {
+      case a: A => Success(a)
```

Comment:
This and other changes are caused by the fact that json-rpc now returned deserialized message. I can probably remove the case with json, though it might be safer to leave as a fallback